### ISSUE TITLE
Problem running script - pdata1

### ISSUE THREAD

**Issue by Zyahko** [skiplagged/pokemongo-python#18]
When I run the script using a dummy account it keeps saying that it was unable to retrieve the pdata1 and just loops logging in through google.

**Comment by aktarer** [comment]
We recently imposed a rate limit of 1 request per second due to high traffic. Try the latest code

**Comment by oleerik** [comment]
Is this rate limit pr IP? :)

**Comment by Zyahko** [comment]
I have updated to the latest code for the skiplagged.py and still some scripts are getting the error for unable to retrieve pdata1

**Comment by aktarer** [comment]
> Is this rate limit pr IP? :)

Yeah, it's an IP limit but please don't get a bunch of IPs and degrade the service for everyone :)

> I have updated to the latest code for the skiplagged.py and still some scripts are getting the error for unable to retrieve pdata1

We have been doing server maintenance and literally just finished. It seems to be working well on our end..

**Comment by oleerik** [comment]
> Yeah, it's an IP limit but please don't get a bunch of IPs and degrade the service for everyone :)
Thank you and of course not. I'll just have to look into how to get a decently up-to-date map over my city now as it might be a bit more tricky with the limitations now in place.

**Comment by Zyahko** [comment]
Ah that might be my issue then if it is IP limit. I have about 7 different scripts run to scan different parts of my city using different dummy accounts.

**Comment by oleerik** [comment]
> Ah that might be my issue then if it is IP limit. I have about 7 different scripts run to scan different parts of my city using different dummy accounts.

Hehe, that is my issue too! :) I might have to look into setting up my own map for now.

**Comment by aktarer** [comment]
Btw, the IP restriction is just temporary while we optimize our systems and will probably go away by early next week.

**Comment by Juice805** [comment]
Are there plans to drop the rate limit?
